gitlab/resource_gitlab_repository_file.go

```diff
@@ -0,0 +1,153 @@
+package gitlab
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabRepositoryFile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabRepositoryFileCreate,
+		Read:   resourceGitlabRepositoryFileRead,
+		Update: resourceGitlabRepositoryFileUpdate,
+		Delete: resourceGitlabRepositoryFileDelete,
+
+		Schema: map[string]*schema.Schema{
+			"project": {
```

Comment:
`project` and `branch` attributes should both have `ForceNew: true` since they are primary lookup keys of the resource.

Reply:
I have added this to `file` too.